riscv-gcc/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+INST_DATE=`git show -s --format=%ci ff21e26eb8c4d55dad7ad0b57e7bd8f7784a60e9 | awk '{split($0,d,"[- ]"); print d[1]d[2]d[3]}'`
+export INST_VERSION=${INST_DATE}-${VERSION}
+export RISCV_PATH=${RISCV_BASE}/${INST_VERSION}
+export PATH=${RISCV_PATH}/bin:${PATH}
+
+git submodule update --init --recursive
+./configure --prefix=${RISCV_PATH} --with-arch=rv32i
```

Comment:
See the comment in https://github.com/librecores/docker-images/blob/master/librecores-ci-riscv/Dockerfile . Likely the default image should include the 64bit architecture and multiple ABIs (now it runs with the default one)